python/paddle/__init__.py

```diff
@@ -11,6 +11,35 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+__all__ = [
```

Comment:
use one item per line format, add annotations to silent style linter errors if needed,
same for other files